.github/workflows/build-soroban-dev.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       core_ref: soroban-preview-10-July20
       core_configure_flags: --disable-tests --enable-next-protocol-version-unsafe-for-production
       go_ref: soroban-v0.0.9.1
-      soroban_tools_ref: v0.9.2
+      soroban_tools_ref: v0.9.4
```

Comment:
this will build soroban rpc from this git tag, that is intent correct? cli is not built in quickstart image, the title of pr refers to 'cli' so wanted to confirm.

Reply:
I thought that the cli is also included in the quickstart docker container. if that's not the case ( please confirm.. ), then I'd like to scrap this PR. We need to release just the cli for now.

Reply:
yes, that is correct, quickstart only builds rpc - https://github.com/stellar/quickstart/blob/master/Makefile#L18, so, sounds like this can be closed.

Reply:
thanks!